### DISCUSSION TITLE
A problem with drawing points in OSG3.4.0

### DISCUSSION THREAD

**Discussion by XianranZ** [openscenegraph/OpenSceneGraph#1277] [Q&A]
I'm experiencing something strange when drawing points with OSG 3.4.0. When I draw only one point, the program does not display the point. When I draw two points, the program displays two points, but the positions of them are wrong. When I draw three points, the program runs correctly and shows three points in the correct position.
Below are my codes and  screenshots of the program running.

Draw one point code
![IMG_20231128_173144.jpg](https://github.com/openscenegraph/OpenSceneGraph/assets/46214062/8f9d549e-d7b7-4dcc-aa85-8c195eb7b978)
The screenshot of program running
![IMG_20231128_173335.jpg](https://github.com/openscenegraph/OpenSceneGraph/assets/46214062/f236a559-1492-4a2a-97d3-7cf892145d0f)

Draw two points code
![IMG_20231128_173205.jpg](https://github.com/openscenegraph/OpenSceneGraph/assets/46214062/d1a3ccc9-6c53-4537-8fcd-703ac9b9cea7)
The screenshot of program running
![IMG_20231128_173239.jpg](https://github.com/openscenegraph/OpenSceneGraph/assets/46214062/1eab0d2d-392e-4b0f-a1d8-eaba0147807d)

Draw three points code
![IMG_20231128_172834.jpg](https://github.com/openscenegraph/OpenSceneGraph/assets/46214062/e6c422d1-0cd3-44e3-819c-6a274d95a487)
The screenshot of program running
![IMG_20231128_173302.jpg](https://github.com/openscenegraph/OpenSceneGraph/assets/46214062/24dbd0aa-47d8-46fc-baf8-202c01d76a29)

The difference between these three cases is only the number of points put in, the rest of the code has not changed. However they are not displaying as expected, why is this?
Looking forward to your answers, thanks.

**Comment by trungns4** [comment]
This should help
  camera->setCullingMode(camera->getCullingMode() &
~osg::CullStack::SMALL_FEATURE_CULLING);

/Tr.

**Reply by XianranZ** [reply]
Ok, thank you very much.
I will try it.
